docs/buildspecs/e4s_testsuite.rst

```diff
@@ -0,0 +1,49 @@
+.. Note:: Please see :ref:`tutorial_setup` before you proceed with this section
+
+.. _buildtest_e4s_testsuite:
+
+Buildtest E4S Testsuite Integration
+===================================
+
+The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
+software products provided by `E4S <https://e4s.io>`_ deployments. 
```

Comment:
since this is first mention of "E4S" it may be better to change `E4S` -> `Extreme Scale Scientific Software Stack (E4S)`. The user may not know what `E4S` is